lib/qx/tool/cli/commands/Compile.js

```diff
@@ -362,8 +362,8 @@ qx.Class.define("qx.tool.cli.commands.Compile", {
       // if only one app set writeIndexHtmlToRoot = true
       if (data.applications.length === 1) {
         let appData = data.applications[0];
-        if (appData.writeIndexHtmlToRoot === undefined) {
-          appData.writeIndexHtmlToRoot = true;
+        if (appData["default"] === undefined) {
+          appData["default"] = true;
         }
       }
```

Comment:
this whole block should be deleted - the code below detects whether default has not been explicitly set (and default app only applies to browser based apps)

Reply:
You are right